Formula/rhit.rb

```diff
@@ -0,0 +1,56 @@
+class Rhit < Formula
+  desc "Nginx log explorer"
+  homepage "https://github.com/Canop/rhit"
+  url "https://static.crates.io/crates/rhit/rhit-1.5.2.crate"
```

Comment:
crate file seems pretty intensive in here, maybe we should ask upstream to create a new release and maintain it?

Reply:
Actually, @Canop does regular releases but only in binary form (https://dystroy.org/rhit/download/), and the git repo doesn't have tags. 🤷

Reply:
ok, we can use the tgz file now. 🎉